.github/actions/run-python-test-set/action.yml

```diff
@@ -40,6 +40,10 @@ inputs:
     description: 'Whether to rerun failed tests'
     required: false
     default: 'false'
+  special_kind:
```

Comment:
Can we come up with a better name? `special_kind` seems really generic term

Reply:
We got rid of that parameter for now.